buildermgr/common.go

```diff
@@ -131,12 +131,12 @@ func updatePackage(fissionClient *crd.FissionClient,
 	}
 
 	// update package spec
-	pkg, err := fissionClient.Packages(metav1.NamespaceDefault).Update(pkg)
```

Comment:
Why is this necessary?  I must be missing something but it looks like it doesn't really change anything...

Reply:
No, this is not necessary. I can rollback to previous version.